### ISSUE TITLE
breaks on some characters on DEK's machine

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#2158]
I just upgraded my macOS machine to the latest version of Sonoma-beta, and it cannot parse some files, e.g. below.  The problem seems to be in the degree sign, and since this is just a comment, I'll change that. 

My guess is that this is just a quirk of my OS (something to do with a change to system 'encoding', perhaps) but I don't see any real benefit in keeping a comment that is just there to help the developers, when an altered comment will be just as helpful.  And maybe other users will have similar problems if they are running Sonoma-beta...

<img width="1095" alt="Screenshot 2023-09-29 at 5 39 47 PM" src="https://github.com/dankelley/oce/assets/99469/c37c850c-96c4-4f10-8289-71267cebabd0">

**Comment by dankelley** [comment]
Hm.  I changed the degree signs to "DEG" (it's just for a comment) and I also change some slanted " marks to non-slanted ones.  Now it builds and checks.

So I thought I'd check all the source files for encoding.  Most are `us-ascii`, but I also see as below.  Just in case there might be a problem, I'll go through those files on the lookout for non-ascii characters.  (I'm not saying I will alter them, though ... I'm just curious.)

```
file -I *R|grep utf-8
adv.sontek.R:       text/plain; charset=utf-8
ctd.saiv.R:         text/plain; charset=utf-8
ctd.sbe.R:          text/plain; charset=utf-8
ctd.ssda.R:         text/plain; charset=utf-8
geod.R:             text/plain; charset=utf-8
rsk.R:              text/plain; charset=utf-8
```

**Comment by dankelley** [comment]
The way to find non-ascii is, in R is e.g.
```R
tools::showNonASCIIfile("rsk.R")
```

**Comment by dankelley** [comment]
Fixed in "develop" commit fc6e7ab25c2cd5df21fdeca944ccf5638dfb6a8d
